src/app/exp-feature-page/exp-feature-page.component.ts

```diff
@@ -1,19 +1,61 @@
 import {
   Component,
-  OnDestroy
+  OnDestroy,
+  OnInit,
+  EventEmitter,
+  Output
 } from '@angular/core';
 
 import 'rxjs/add/operator/map';
+import { Notifications, Notification, NotificationType } from 'ngx-base';
+import { GettingStartedService } from '../getting-started/services/getting-started.service';
 
 @Component({
   selector: 'f8-exp-feature-page',
   templateUrl: './exp-feature-page.component.html'
 })
-export class ExpFeaturePageComponent implements OnDestroy {
+export class ExpFeaturePageComponent implements OnInit, OnDestroy {
 
-  constructor() {
+  enableFeatures: boolean;
+
+  @Output() onExperimentalFeaturesEnabled = new EventEmitter<boolean>();
+
+  constructor(private gettingStartedService: GettingStartedService,
+              private notifications: Notifications) {
+  }
+
+  ngOnInit() {
+    this.enableFeatures = false;
   }
 
   ngOnDestroy() {
   }
+
+  activateExperimentalFeatures() {
+    let profile = this.gettingStartedService.createTransientProfile();
+
+    if (!profile.contextInformation) {
+      profile.contextInformation = {};
+    }
+
+    if (!profile.contextInformation.experimentalFeatures) {
+      profile.contextInformation.experimentalFeatures = {};
+    }
+    profile.contextInformation.experimentalFeatures["enabled"] = true;
+
+    this.gettingStartedService.update(profile).subscribe(() => {
+      // reset boolean in case they come back in here later
+      this.enableFeatures = false;
+      this.onExperimentalFeaturesEnabled.emit(true);
+      this.notifications.message({
+        message: `Profile updated!`,
```

Comment:
almost wondering if this should be 'Experimental Feature Enabled', or something along those lines?